client-proxy/consumer/consumer.go

```diff
@@ -0,0 +1,24 @@
+package consumer
+
+import (
+	"github.com/OpenLineage/OpenLineage/client-proxy/database"
+	"github.com/OpenLineage/OpenLineage/client-proxy/transports"
+)
+
+type Consumer struct {
+	db        database.IDatabase
+	transport transports.ITransport
+}
+
+type IConsumer interface {
+	Run()
+}
+
+func (consumer *Consumer) Run() {
+	// Not yet implemented
```

Comment:
With non-implemented functions, I suggested adding:

```
panic("Consumer.Run() not implemented")
```

Reply:
From what I understood, the `panic` will terminate the process immediately. Developers would want to try run the app multiple times of development iteration and this will block the testing. Of course the unit tests could run with `panic` here but that doesn't mean we discourage doing manual testing.

Reply:
Ahh, I see. You may want to log that this application has started, but otherwise 👍